components/FishForm.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
+import { updateFishEntry } from '../actions'
+import { connect } from "react-redux";
+import RNPickerSelect from 'react-native-picker-select';
+
+class FishForm extends Component {
+  state = {
+    species: '',
+    image: '',
+    length: '',
+    weight: '',
+		location: '',
```

Comment:
A little spacing issue that we can clean up.

Reply:
Yeah. I think it is my VS code that makes those spaces.  I don't know how to set it up so that it doesn't have that much spacing while I tab.  I am new to VS code.  I agree that we will need to clean up those white spaces to make it look cleaner.